<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="7" passed="3" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-09-08T15:41:53 IST" name="Suite" finished-at="2025-09-08T15:42:36 IST" duration-ms="42219">
    <groups>
      <group name="dataProvider">
        <method signature="TC003_DDTLoginTest.verify_Login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_DDTLoginTest@6e9175d8]" name="verify_Login" class="testCases.TC003_DDTLoginTest"/>
      </group> <!-- dataProvider -->
    </groups>
    <test started-at="2025-09-08T15:41:53 IST" name="Test" finished-at="2025-09-08T15:42:36 IST" duration-ms="42219">
      <class name="testCases.TC003_DDTLoginTest">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_DDTLoginTest@6e9175d8]" started-at="2025-09-08T15:41:54 IST" name="setup" finished-at="2025-09-08T15:41:58 IST" duration-ms="4370" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[window]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verify_Login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_DDTLoginTest@6e9175d8]" started-at="2025-09-08T15:41:59 IST" name="verify_Login" data-provider="LoginData" finished-at="2025-09-08T15:42:01 IST" duration-ms="1591" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[abababnc@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1234]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_Login -->
        <test-method signature="verify_Login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_DDTLoginTest@6e9175d8]" started-at="2025-09-08T15:42:01 IST" name="verify_Login" data-provider="LoginData" finished-at="2025-09-08T15:42:03 IST" duration-ms="1802" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[aka@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1234]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_Login -->
        <test-method signature="verify_Login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_DDTLoginTest@6e9175d8]" started-at="2025-09-08T15:42:03 IST" name="verify_Login" data-provider="LoginData" finished-at="2025-09-08T15:42:18 IST" duration-ms="15148" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[aka@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1234567]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Invalid]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at testCases.TC003_DDTLoginTest.verify_Login(TC003_DDTLoginTest.java:50)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:569)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_Login -->
        <test-method signature="verify_Login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_DDTLoginTest@6e9175d8]" started-at="2025-09-08T15:42:18 IST" name="verify_Login" data-provider="LoginData" finished-at="2025-09-08T15:42:34 IST" duration-ms="15637" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[aka@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1234567]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Invalid]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at testCases.TC003_DDTLoginTest.verify_Login(TC003_DDTLoginTest.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:569)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_Login -->
        <test-method signature="verify_Login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC003_DDTLoginTest@6e9175d8]" started-at="2025-09-08T15:42:34 IST" name="verify_Login" data-provider="LoginData" finished-at="2025-09-08T15:42:35 IST" duration-ms="1623" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[abababnc@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1234]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_Login -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC003_DDTLoginTest@6e9175d8]" started-at="2025-09-08T15:42:36 IST" name="tearDown" finished-at="2025-09-08T15:42:36 IST" duration-ms="189" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC003_DDTLoginTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
